client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -319,11 +319,19 @@
 
             stripHtml(diff);
             autosave.stop(item);
-            return api.save('archive', item, diff).then(function(_item) {
-                item._autosave = null;
-                item._locked = lock.isLocked(item);
-                return item;
-            });
+
+            if (_.size(diff) > 0) {
+                return api.save('archive', item, diff).then(function(_item) {
+                    item._autosave = null;
+                    item._locked = lock.isLocked(item);
+                    return item;
+                });
+            } else {
+                var d = $q.defer();
+                d.resolve(origItem);
+
+                return d.promise;
```

Comment:
you can use `return $q.when(origItem);` which returns a promise that's resolved with given value